relationships/candidates/AWSMSKTOPIC2.yml

```diff
@@ -0,0 +1,20 @@
+category: AWSMSKTOPIC2
```

Comment:
Is there any reason to create a new category rather than updating `AWSMSKTOPIC`?
`AWSMSKTOPIC` category is not used in any relationship rule, so it shouldn't be a problem.